tests/rdfc10/test058-rdfc10map.json

```diff
@@ -0,0 +1,5 @@
+{
+  "e3": "c14n0",
+  "e0": "c14n1",
+  "e1": "c14n2"
+}
```

Comment:
I think this file should be removed because test058 has ambiguous (duplicated) `issued identifiers map`s.